src/utils/util.ts

```diff
@@ -0,0 +1,75 @@
+import { toast } from "react-toastify";
+import * as blake from "blakejs";
+import { bech32m } from "bech32";
+
+export const DECIMAL = 100000000000;
+
+export const encodeToHash = (data: string) => {
+  const hash = blake.blake2bHex(data, undefined, 32);
+  return hash;
+};
+
+export const encode = (prefix: string, data: ArrayLike<number>) => {
```

Comment:
is this function actually used? Same for functions below. All dead code should be removed.